src/CONST.js

```diff
@@ -571,6 +571,7 @@ const CONST = {
     },
     JSON_CODE: {
         SUCCESS: 200,
+        DUPLICATE_RECORD: 400,
```

Comment:
@johnmlee101 Can we make use of this JSON_CODE here, is this code used for any other purposes?

Reply:
Yeah there are a few other places in the back-end that could throw a 400. I'd rather we capture the full response to check if it mentions the duplicate entry, rather than assign the code entirely.